language/en/interface.json

```
@@ -1665,6 +1667,10 @@
 				"title": "%{role1}/%{role2}",
 				"description": "%{role1}\n\nand/or\n\n%{role2}"
 			}
+		},
+		"quicksharetotarget": {
+			"noTarget": "No target selected",
+			"shareTo": "Share to:"
```

Use string interpolation inside the translation entry rather than string concatenation, not all languages have the same subject-object-verb ordering.
```suggestion
			"shareTo": "Share to: %{playerColor}%{player}"
```